allennlp/modules/softmax_loss.py

```diff
@@ -22,6 +23,20 @@ def __init__(self, num_words: int, embedding_dim: int) -> None:
 
     def forward(self, embeddings: torch.Tensor, targets: torch.Tensor) -> torch.Tensor:
 
+        """
+        # Parameters
+
+        embeddings : `torch.Tensor`
+            A tensor of shape `(sequence_length, embedding_dim)`
+        targets : `torch.Tensor`
+            A tensor of shape `(batch_size, )`
+
+        # Returns
+
+        loss : `torch.FloatTensor`
+            A scalar loss to be optimized.
+        """
+
```

Comment:
Just removing extra empty lines

```suggestion
        """
        # Parameters

        embeddings : `torch.Tensor`
            A tensor of shape `(sequence_length, embedding_dim)`
        targets : `torch.Tensor`
            A tensor of shape `(batch_size, )`

        # Returns

        loss : `torch.FloatTensor`
            A scalar loss to be optimized.
        """
```